uber/models.py

```diff
@@ -961,6 +970,7 @@ class Attendee(MagModel, TakesPaymentMixin):
     no_shirt          = relationship('NoShirt', backref=backref('attendee', load_on_pending=True), uselist=False)
     admin_account     = relationship('AdminAccount', backref=backref('attendee', load_on_pending=True), uselist=False)
     food_restrictions = relationship('FoodRestrictions', backref=backref('attendee', load_on_pending=True), uselist=False)
+    watchlist_entry   = relationship('WatchList', backref=backref('attendee', load_on_pending=True), uselist=False)
```

Comment:
Are you sure you want ``uselist=False`` here?  If someone registered twice, wouldn't both registrations point to the same ``Watchlist`` entry?  This seems like a many-to-one relationship.